x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/find_threshold_signals.ts

```diff
@@ -138,7 +138,7 @@ export const findThresholdSignals = async ({
     logger,
     // @ts-expect-error refactor to pass type explicitly instead of unknown
     filter,
-    pageSize: 1,
+    pageSize: 0,
```

Comment:
@madirey is this changed intended for `master` as well?

Reply:
@spong Yeah, we don't actually use those results... just the aggs. It was unnecessary since the multi-aggs code was added and the codepath was generalized.